WebView/app/src/main/java/com/android/samples/webviewdemo/MainActivity.kt

```diff
@@ -88,10 +87,10 @@ class MainActivity : AppCompatActivity() {
         // Enable Javascript
         binding.webview.settings.javaScriptEnabled = true
 
-        // Connect to Javascript Interface
+        //Connect to WebApp Interface
         binding.webview.addJavascriptInterface(WebAppInterface(this), "Weather")
 
-        // Load the content
-        binding.webview.loadUrl("https://gcoleman799.github.io/Asset-Loader/index.html")
+        //Load the content
```

Comment:
Please put a space after "//" (here and elsewhere)

Reply:
Ok